packages/generator/src/artifacts/types/generated.ts

```diff
@@ -28,8 +28,22 @@ export function getGeneratedTypes(ctx: Context) {
     system: rewriteImports(
       match(ctx.jsx.styleProps)
         .with('all', () => system.content)
-        .with('minimal', () => system.content.replace(jsxStyleProps, 'export type JsxStyleProps = WithCss'))
-        .with('none', () => system.content.replace(jsxStyleProps, 'export type JsxStyleProps = {}'))
+        .with('minimal', () =>
```

Comment:
this seems to solve only part of the problem, the omitted props still need to be applied when used with `html*` prefix

Reply:
good catch ! fixed and added a test there

https://github.com/chakra-ui/panda/pull/1552/commits/35f75ead73e6056a913de54843182805e17e346e#diff-2e5a346ed5ecd3ed6d409a1b4696b1b01b68351f077dbbf3efba8fa4a931669eR288-R303